AndroidAnnotations/functional-test-1-5/src/main/java/org/androidannotations/test15/ActivityWithServices.java

```diff
@@ -121,6 +125,9 @@
 	@SystemService
 	DropBoxManager dropBoxManager;
 	
+	// @SystemService
+	// EthernetManager ethernetManager; // hidden API
```

Comment:
Yeah, please see this
https://github.com/android/platform_frameworks_base/blob/master/core/java/android/content/Context.java

We support internal services since 1fee3bbc. These are useful for
developers of some kind of core apps, but i cannot find the issue.